packages/nodes-base/nodes/HttpRequest/V3/Description.ts

```diff
@@ -1167,6 +1169,10 @@ export const mainProperties: INodeProperties[] = [
 			},
 		],
 	},
+	...optimizeResponseProperties.map((x) => ({
```

Comment:
Please avoid using x as a variable name, please use the actual property name, because it's clearer.